.github/scripts/publish-addon.sh

```diff
@@ -2,8 +2,7 @@
 set -e
 # this script publishes the addon zip file to the unofficial python2/3 kodi repo 
 
-VERSION=$1
```

Comment:
This Version variable is still used in line 25. will it still work even it is removed?